src/main/java/Duke.java

```diff
@@ -1,10 +1,377 @@
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.io.*;
```

Comment:
Maybe the wild card import can be replaced with specific importation of classes?